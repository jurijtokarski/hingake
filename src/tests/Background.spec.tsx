import { h } from "preact";
import { shallow, ShallowWrapper } from "enzyme";

import Component from "../app/Background";

const ChildrenMock = () => <div></div>;

describe("Index", () => {
  let component: ShallowWrapper;

  beforeAll(() => {
    component = shallow(
      <Component>
        <ChildrenMock />
      </Component>
    );
  });

  test("matches snapshot", () => {
    expect(component).toMatchSnapshot();
  });

  test("renders children", () => {
    expect(component.find(ChildrenMock).length).toBe(1);
  });

  test("renders className", () => {
    expect(component.hasClass("background")).toBe(true);
  });
});
