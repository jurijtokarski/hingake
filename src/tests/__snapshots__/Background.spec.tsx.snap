// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background matches snapshot 1`] = `ShallowWrapper {}`;
